tests/smcp_gold_panda.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"maistra/util"
+	"testing"
+	"time"
+
+	"istio.io/pkg/log"
+)
+
+func cleanupGoldPandaSMCP(namespace string) {
+	log.Info("# Cleanup ...")
+	util.KubeDelete(namespace, goldPandaSMCP, kubeconfig)
+	time.Sleep(time.Duration(waitTime*8) * time.Second)
+	// avoid namespace recreation for downstream service account settings
+}
+
+func TestGoldPandaSMCP(t *testing.T) {
+	defer cleanuGoldPandaSMCP("istio-gold-panda")
```

Comment:
@gbaufake there is a typo in line 33 . `cleanu` should be `cleanup`

Reply:
I already fixed that